src/model/user-settings/UserSettings.ts

```diff
@@ -111,14 +111,14 @@ export class UserSettings implements IUserSettings {
     let scroll =
       (await this.getProperty(ReadiumCSS.SCROLL_KEY)) != null
         ? ((await this.getProperty(ReadiumCSS.SCROLL_KEY)) as Switchable).value
-        : this.verticalScroll;
+        : UserSettings.parseScrollSetting(this.verticalScroll);
```

Comment:
I think `parseScrollSetting` should only be needed when you're taking one of the input settings and transforming it into a boolean. I believe `this.verticalScroll` should always already be a boolean though. Leaving this as `this.verticalScroll` should work, along with at line 121 below

Reply:
yeah that's what i thought as well, but for some reason it didn't work, i'll try again though

Reply:
yes that works, reverting back.